Build fails since #2394

- Build fails since https://github.com/espeak-ng/espeak-ng/pull/2394#issuecomment-4326658525

```c

s/build espeak-ng
CLEAN      espeak-ng
    *      Removing /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/build/espeak-ng-8abc28a0683a70f3adae3be04dc66fe5a0edde92 ...
    *      Removing /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/qa_checks/espeak-ng-* ...
UNPACK      espeak-ng
    APPLY PATCH (common)      packages/audio/espeak-ng/patches/0001-Fix-stack-buffer-overflows-in-number-and-clause-form.patch
patching file src/libespeak-ng/numbers.c
patching file src/libespeak-ng/readclause.c
    APPLY PATCH (common)      packages/audio/espeak-ng/patches/0001-Replace-sizeof-path_home-N-buffer-sizing-with-N_PATH.patch
patching file src/libespeak-ng/compiledata.c
patching file src/libespeak-ng/compiledict.c
patching file src/libespeak-ng/compilembrola.c
patching file src/libespeak-ng/dictionary.c
patching file src/libespeak-ng/langopts.c
patching file src/libespeak-ng/soundicon.c
patching file src/libespeak-ng/speech.c
patching file src/libespeak-ng/speech.h
patching file src/libespeak-ng/synth_mbrola.c
patching file src/libespeak-ng/synthdata.c
patching file src/libespeak-ng/voices.c
    APPLY PATCH (common)      packages/audio/espeak-ng/patches/0001-voices-fix-unsafe-strcat-into-fixed-size-voice_ident.patch
patching file src/libespeak-ng/voices.c
    APPLY PATCH (common)      packages/audio/espeak-ng/patches/cm.patch
patching file src/libespeak-ng/CMakeLists.txt
BUILD      espeak-ng (host)
    TOOLCHAIN      cmake (auto-detect)
Executing (host): cmake -GNinja -DCMAKE_TOOLCHAIN_FILE=/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/etc/cmake-x86_64-linux-gnu.conf -DCMAKE_INSTALL_PREFIX=/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCOMPILE_INTONATIONS=OFF -DENABLE_TESTS=OFF -DESPEAK_COMPAT=OFF -DUSE_LIBSONIC=ON /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/build/espeak-ng-8abc28a0683a70f3adae3be04dc66fe5a0edde92
-- The C compiler identification is GNU 16.0.1
-- The CXX compiler identification is GNU 16.0.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Looking for mkstemp
-- Looking for mkstemp - found
-- Looking for nbtool_config.h
-- Looking for nbtool_config.h - not found
-- Looking for optreset
-- Looking for optreset - not found
-- Looking for sys/endian.h
-- Looking for sys/endian.h - not found
-- Looking for iswblank
-- Looking for iswblank - found
-- Configuration:
--   shared: OFF
--   mbrola: OFF (MBROLA_BIN-NOTFOUND)
--   libsonic: ON (/var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/lib/libsonic.a /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/include)
--   libpcaudio: OFF (PCAUDIO_LIB-NOTFOUND PCAUDIO_INC-NOTFOUND)
--   klatt: ON
--   speech-player: ON
--   async: ON
-- Configuring done (0.3s)
-- Generating done (0.0s)
-- Build files have been written to: /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/build/espeak-ng-8abc28a0683a70f3adae3be04dc66fe5a0edde92/.x86_64-linux-gnu
Executing (host): ninja 
[47/47] Link espeak-ng to compat names
[1/2] Install the project...
-- Install configuration: "Release"
CMake Error at cmake_install.cmake:54 (file):
  file cannot create directory: /pkgconfig.  Maybe need administrative
  privileges.


FAILED: [code=1] CMakeFiles/install.util 
cd /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/build/espeak-ng-8abc28a0683a70f3adae3be04dc66fe5a0edde92/.x86_64-linux-gnu && /var/media/DATA/home-rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-13.0-devel/toolchain/bin/cmake -P cmake_install.cmake
ninja: build stopped: subcommand failed.

```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,6 +26,7 @@ if (WITH_COVERAGE)
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -fprofile-instr-generate -fcoverage-mapping")
 endif()
 
+include(GNUInstallDirs)
 include(cmake/deps.cmake)
 include(cmake/config.cmake)
 add_subdirectory(src)

--- a/cmake/package.cmake
+++ b/cmake/package.cmake
@@ -21,8 +21,8 @@ set(VERSION ${PROJECT_VERSION})
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
 set(exec_prefix ${CMAKE_INSTALL_PREFIX})
-set(libdir "\${exec_prefix}/lib")
-set(includedir "\${prefix}/include")
+set(libdir ${CMAKE_INSTALL_FULL_LIBDIR})
+set(includedir ${CMAKE_INSTALL_FULL_INCLUDEDIR})
 
 configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/${PROJECT_NAME}.pc.in

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -14,7 +14,7 @@ target_link_libraries(
 set_target_properties(
   espeak-ng-bin PROPERTIES
   MACOSX_RPATH ON
-  INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib"
+  INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}"
 )
 if (MINGW)
   target_link_options(espeak-ng-bin PRIVATE "-static-libstdc++" "-static")

--- a/src/libespeak-ng/CMakeLists.txt
+++ b/src/libespeak-ng/CMakeLists.txt
@@ -40,7 +40,7 @@ add_library(espeak-ng
 set_target_properties(
   espeak-ng PROPERTIES
   MACOSX_RPATH ON
-  INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib"
+  INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}"
 )
 
 target_include_directories(espeak-ng BEFORE PRIVATE $<TARGET_PROPERTY:espeak-include,INTERFACE_INCLUDE_DIRECTORIES>)

